### DISCUSSION TITLE
Does the Vega-lite Having a fixed Y-axis?

### DISCUSSION THREAD

**Discussion by PonBalaji** [vega/vega-lite#7831] [Q&A]
Hi ,
I have used Vega-lite in my project.
On my first image the Y-axis on the left side.
But when i scrolling from left to right the Y-axis is hidden from the screen(second image).
why the Y-axis is not so fixed on the left-side.

![1](https://user-images.githubusercontent.com/60576869/144445174-ef7ceb19-28cc-4933-881a-1327030e3443.png)
![2](https://user-images.githubusercontent.com/60576869/144445190-5b828bec-61f6-426f-b0cf-47cb308bc920.png)

**Comment by kanitw** [comment]
You can make the chart pannable like this: https://vega.github.io/vega-lite/examples/selection_translate_scatterplot_drag.html
